schema/release-schema.json

```diff
@@ -2283,6 +2293,26 @@
           ]
         }
       }
+    },
+    "Link": {
+      "title": "Link",
+      "description": "A typed connection to a related resource.",
+      "type": "object",
+      "properties": {
+        "rel": {
+          "title": "Link Relation Type",
```

Comment:
Use sentence case for title properties.

Reply:
Done in https://github.com/open-contracting/standard/pull/1326/commits/8d57e733867f03087514cd804cf646d6bf80ea29